clib/src/main/scala/scala/scalanative/libc/linux/epoll.scala

```diff
@@ -0,0 +1,95 @@
+package scala.scalanative
+package libc
+package linux
+
+/*
+ * https://man7.org/linux/man-pages/man2/epoll_wait.2.html
+ * https://man7.org/linux/man-pages/man2/epoll_create1.2.html
+ */
+
+import unsafe._
+import unsigned._
+
+@extern
+@define("__SCALANATIVE_C_EPOLL")
```

Comment:
This definition is not used anywhere currently. The concept of using `@define` for bindings is used to skip compilation of related glue layer if it's not actually used, so that we reduce the the possibility that unused glue layer in C won't compile. 
So epoll.c should have additional guard `if defined(__SCALANATIVE_C_EPOLL)` , however typically there's also the additional include to compile it always useful when setting up IDE support with clangd, eg. 
```
#if defined(SCALANATIVE_COMPILE_ALWAYS) || defined(__SCALANATIVE_C_EPOLL)
```

see fcntl.c below for example